libraries/xcode_command_line_tools.rb

```diff
@@ -154,7 +154,7 @@ def attach
     # @return [void]
     #
     def install
-      execute %|installer -package "$(find '#{mount_path}' -name *.mpkg)" -target "/"|
+      execute %|installer -allowUntrusted -package "$(find '#{mount_path}' -name *.mpkg)" -target "/"|
```

Comment:
This is executed only on OSX 10.7 and 10.8 (when CLTools are installed from dmg). Unfortunately, I don't have an OSX 10.8 box to check if it needs -allowUtrusted flag. Maybe we can leave the code as is and just add the comment?